### PR TITLE
✨ Add datadog unit/reactor charts

### PR DESCRIPTION
**Pull request by b3j0f**
This PR aims to add datadog unit/reactor charts in order to fetch/push datadog metrics

- [x] 📦 Unit chart
- [x] 📦 Reactor chart

### CODE REVIEW

File: incubator/kubirds-datadog-reactor/Chart.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: kubirds-datadog-reactor
+description: Parametrable Datadog Reactor for Kubirds
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```
Comment (linkdd) [inline incubator/kubirds-datadog-reactor/Chart.yaml:6]: Is datadog in version `1.16.0`?
Is the docker image used in the reactor provided by this chart in version `1.16.0`?

File: incubator/kubirds-datadog-reactor/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+helm.sh/chart: {{ include "kubirds-datadog-reactor.chart" . }}
+{{ include "kubirds-datadog-reactor.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
```
Comment (linkdd) [inline incubator/kubirds-datadog-reactor/templates/_helpers.tpl:46]: Used only for `Deployment` and `StatefulSet` resources. Not needed here.

File: incubator/kubirds-datadog-reactor/templates/reactor.yaml [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+apiVersion: kubirds.com/v1
+kind: Reactor
+metadata:
+  name: {{ include "kubirds-datadog-reactor.fullname" . }}
+  labels:
+    {{- include "kubirds-datadog-reactor.labels" . | nindent 4 }}
+    {{ if .Values.labels }}
+    {{ .Values.labels | toYaml | nindent 4 }}
+    {{ end}}
+spec:
```
Comment (linkdd) [inline incubator/kubirds-datadog-reactor/templates/reactor.yaml:11]: New in Kubirds 2.0.0-alpha1:

```yaml
serviceAccountName: {{ include "kubirds-datadog-reactor.serviceAccountName" . }}
```

Every unit and reactor now has a service account defaulting to `default`. The helm chart should override this.

File: incubator/kubirds-datadog-reactor/values.yaml [hunk 4]
```diff
@@ -0,0 +1,23 @@
+---
```
Comment (linkdd) [inline incubator/kubirds-datadog-reactor/values.yaml:1]: Missing `serviceAccount` configuration

File: incubator/kubirds-datadog-unit/Chart.yaml [hunk 5]
```diff
@@ -0,0 +1,6 @@
+apiVersion: v2
+name: kubirds-datadog-unit
+description: Parametrable Datadog Unit for Kubirds
+type: application
+version: 0.1.0
+appVersion: "1.16.0"
```
Comment (linkdd) [inline incubator/kubirds-datadog-unit/Chart.yaml:6]: Same as above, this is not version `1.16.0`

File: incubator/kubirds-datadog-unit/templates/_helpers.tpl [hunk 6]
```diff
@@ -0,0 +1,62 @@
...
+helm.sh/chart: {{ include "kubirds-datadog-unit.chart" . }}
+{{ include "kubirds-datadog-unit.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
```
Comment (linkdd) [inline incubator/kubirds-datadog-unit/templates/_helpers.tpl:46]: Same as above, this is useless

File: incubator/kubirds-datadog-unit/templates/unit.yaml [hunk 7]
```diff
@@ -0,0 +1,34 @@
...
+apiVersion: kubirds.com/v1
+kind: Unit
+metadata:
+  name: {{ include "kubirds-datadog-unit.fullname" . }}
+  labels:
+    {{- include "kubirds-datadog-unit.labels" . | nindent 4 }}
+    {{ if .Values.labels }}
+    {{ .Values.labels | toYaml | nindent 4 }}
+    {{ end}}
+spec:
```
Comment (linkdd) [inline incubator/kubirds-datadog-unit/templates/unit.yaml:11]: Same as above, you can add `serviceAccountName` here.

File: incubator/kubirds-datadog-unit/values.yaml [hunk 8]
```diff
@@ -0,0 +1,31 @@
+---
```
Comment (linkdd) [inline incubator/kubirds-datadog-unit/values.yaml:1]: Same as above, missing service account configuration

**Review by linkdd — CHANGES_REQUESTED** [review]

File: incubator/kubirds-datadog-reactor/Chart.yaml (hunk 1 above)
Comment (b3j0f) [reply incubator/kubirds-datadog-reactor/Chart.yaml:6]: I don't know which version to use

For example, this chart permit to use both v1/v2 api
Datadog agent use the v7

Help please ^^'

File: incubator/kubirds-datadog-reactor/templates/_helpers.tpl (hunk 2 above)
Comment (b3j0f) [reply incubator/kubirds-datadog-reactor/templates/_helpers.tpl:46]: Common/Selector labels are useless ?

**Pull request closed without merging**